src/main/java/com/google/codeu/servlets/UserListServlet.java

```diff
@@ -34,3 +34,4 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
   }
```

Comment:
It would be great if there was a comment for this function that explained the format of the JSON that it's going to return. Right now it's hard to know what it looks like without starting the server and connecting to that endpoint.

Reply:
resolved. See my new commit to this branch